$OpenBSD: patch-src_test1_c,v 1.4 2010/04/02 20:07:11 sthen Exp $

Fix test on big endian arches.  Bug filed upstream at
http://www.sqlite.org/cvstrac/tktview?tn=3621

--- src/test1.c.orig	Tue Jan  5 00:02:41 2010
+++ src/test1.c		Wed Jan 20 11:15:08 2010
@@ -1199,7 +1199,7 @@ static int sqlite3_mprintf_int(
   int argc,              /* Number of arguments */
   char **argv            /* Text of each argument */
 ){
-  int a[3], i;
+  unsigned int a[3], i;
   char *z;
   if( argc!=5 ){
     Tcl_AppendResult(interp, "wrong # args: should be \"", argv[0],
